dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -184,20 +203,22 @@ CommandBarFlyout::CommandBarFlyout()
                             m_isClosingAfterCloseAnimation = false;
                         });
                 }
+                else
+                {
+                    // If we don't have an animation, close the command bar and thus it's subflyouts.
+                    commandBar->IsOpen(false);
+                }
 
-                // Close commandbar and thus other associated flyouts
-                commandBar->IsOpen(false);
-
-                //CommandBarFlyoutCommandBar.Closed will be called when
-                //clicking the more (...) button, we clear the translations
-                //here
-                commandBar->ClearShadow();
+                //Drop shadows do not play nicely with clip animations, if we are using both, clear the shadow
+                if (SharedHelpers::Is21H1OrHigher() && commandBar->OpenAnimationKind() == CommandBarFlyoutOpenCloseAnimationKind::Clip)
```

Comment:
Do we need to do a similar check for the `RemoveDropShadow` call up above in this same `Closing` function?

Reply:
Yes, you are right.

Reply:
Actually, I was right :).  The primary items shadow had to be placed on the flyout presenter to escape the clip, which means we aren't able to fade out the shadow while we close like we can with the secondary items, so we have to remove the drop shadow in all cases.